src/ImageSharp/Formats/Gif/GifEncoder.cs

```diff
@@ -30,6 +30,14 @@ public sealed class GifEncoder : IImageEncoder, IGifEncoderOptions
         /// </summary>
         public IQuantizer Quantizer { get; set; } = new OctreeQuantizer();
 
+        /// <summary>
+        /// Gets or sets a value indicating whether pixels that stay the same between two frames should be reused
+        /// by forcing the <see cref="DisposalMethod"/> of all frames to <see cref="DisposalMethod.NotDispose"/>
+        /// and making the repeated pixel transparent. This improves GIF quality when pixels are
+        /// repeated across frames.
```

Comment:
This is kind of verbose, but I think this is all relevant information to the user so not sure how to improve.